data/winevt_features.yaml

```diff
@@ -648,3 +688,295 @@ system_7045_v0:
+  provider_identifier:
+    - '{EF1CC15B-46C1-414E-BB95-E76B077BD51E}'
+  event_identifier: 3
+  event_version: 2
+  references:
+  mapping:
+    - name: job_title
+      string_index: 0
+    - name: job_id
+      string_index: 1
+    - name: job_owner
+      string_index: 2
+    - name: process_path
+      string_index: 3
+    - name: process_id
+      string_index: 4
+
+### Version 3 is used in win2k19 and win2k22.
+bits_client_3_v3:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{ef1cc15b-46c1-414e-bb95-e76b077bd51e}'
+  event_identifier: 3
+  event_version: 3
+  references:
+  mapping:
+    - name: job_title
+      string_index: 0
+    - name: job_id
+      string_index: 1
+    - name: job_owner
+      string_index: 2
+    - name: process_path
+      string_index: 3
+    - name: process_id
+      string_index: 4
+    - name: client_process_start_key
+      string_index: 5
+
+## EventID 4: BITS transfer job is complete.
+### Version 1 is used by win2k8 to win2k22
+bits_client_4_v1:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{ef1cc15b-46c1-414e-bb95-e76b077bd51e}'
+  event_identifier: 4
+  event_version: 1
+  references:
+  mapping:
+    - name: user
+      string_index: 0
+    - name: job_title
+      string_index: 1
+    - name: job_id
+      string_index: 2
+    - name: job_owner
+      string_index: 3
+    - name: file_count
+      string_index: 4
+    - name: bytes_transferred
+      string_index: 5
+      aliases:
+        - bytes_out
+    - name: bytes_transferred_from_peer
+      string_index: 6
+      aliases:
+        - bytes_in
+
+## Event ID 59: BITS started the transfer job.
+### Version 1 is used in win2k16, win2k19, and win2k22
+bits_client_59_v1:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{ef1cc15b-46c1-414e-bb95-e76b077bd51e}'
+  event_identifier: 59
+  event_version: 1
+  references:
+    - https://www.mandiant.com/resources/blog/attacker-use-of-windows-background-intelligent-transfer-service
+  mapping:
+    - name: transfer_id
+      string_index: 0
+    - name: name
+      string_index: 1
+    - name: id
+      string_index: 2
+    - name: url
+      string_index: 3
+    - name: peer
+      string_index: 4
+    - name: file_time
+      string_index: 5
+    - name: file_length
+      string_index: 6
+    - name: bytes_total
+      string_index: 7
+    - name: bytes_transferred
+      string_index: 8
+      aliases:
+        - bytes_out
+    - name: bytes_transferred_from_peer
+      string_index: 9
+      aliases:
+        - bytes_in
+
+## Event ID 60: BITS stopped transferring the transfer job.
+bits_client_60_v1:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{ef1cc15b-46c1-414e-bb95-e76b077bd51e}'
+  event_identifier: 60
+  event_version: 1
+  references:
+    - https://www.mandiant.com/resources/blog/attacker-use-of-windows-background-intelligent-transfer-service
```

Comment:
The mandiant blog does not contain any mention of event ID 60 nor the field mapping. Do you happen to have another reference?

Reply:
https://community.microfocus.com/cfs-file/__key/communityserver-wikis-components-files/00-00-00-00-23/MSWinBITSClientEvtLogs.pdf